lib/smart_proxy_ansible/playbooks_reader.rb

```diff
@@ -0,0 +1,46 @@
+
+module Proxy
+  module Ansible
+
+    class PlaybooksReader
```

Comment:
Even though ruby does not care about how files are named, it is good to keep the name of file and class it contains in sync as it greatly helps with finding things. Since class is ReaderHelper now, it is expected file name would be reader_helper.rb